apps/nextjs/package.json

```diff
@@ -24,7 +25,9 @@
     "@trpc/next": "^10.40.0",
     "@trpc/react-query": "^10.40.0",
     "@trpc/server": "^10.40.0",
-    "next": "^13.5.4",
+    "flowbite": "^2.3.0",
```

Comment:
Overall IDK how I feel about adding an extra dependency in "flowbite" but once we get shadcn in this can be replaced. Just want to lower that bundle size, yk.